pokemon-app/src/components/Search.js

```diff
@@ -13,65 +13,70 @@ const Search = ({ getPokeByNameOrIdAPI, isEnemy, setIsEnemy }) => {
 
   const onSubmit = (e) => {
     e.preventDefault();
-
     const pokeLowerCase = pokemonSearchValue.toLowerCase();
 
     setHasSelected(true);
     getPokeByNameOrIdAPI(pokeLowerCase);
   };
+  const removePokemon = () => {
+    if(enemyPokemon === null){
+      if(pokemon !== null){
+        setPokemon(null);
+      }
+    } else {
+      setEnemyPokemon(null);
+      setHasEnemySubmit(false);
+      setIsEnemy(false)
+    }
+  }
 
   return (
     <>
       <form className="searchbar" onSubmit={onSubmit}>
-        <button type="submit" id="searchButton"></button>
+        <div className="remove-poke-title">Remove Pokemon</div>
+        <button type="button" id="removeButton" onClick={() => removePokemon()}>x</button>
         <input
           type="text"
           name="pokemonsearch"
           placeholder="Search for a pokemon"
           value={pokemonSearchValue}
           onChange={onChange}
         />
-      </form>
 
-      <div className="twosearch-pokemon">
-        <div className="twosearch-pokemon__title">
-          <p className="paragraph-versus">Versus</p>
-        </div>
+        <button type="submit" className="search-button">Search</button>
```

Comment:
I notice that you're mixing controlled form logic with uncontrolled form logic.  Which one are you trying to implement?